operators/samples/instanceRS.yml

```diff
@@ -0,0 +1,11 @@
+apiVersion: crownlabs.polito.it/v1alpha2
```

Comment:
Please, delete all additional samples from this PR. I understand they are useful for your tests, but they shouldn't pollute the upstream repo.

Reply:
Fixed in [adb1ac6](https://github.com/netgroup-polito/CrownLabs/pull/821/commits/adb1ac6aac726571d8ebfd9ef95f2653920a0d87).